lldb/test/Shell/SwiftREPL/lit.local.cfg

```diff
@@ -2,3 +2,14 @@ config.suffixes = ['.test']
 
 if 'lldb-repro' in config.available_features:
   config.unsupported = True
+
+def check_exists(path):
+  import os
+  if not os.path.isfile(path):
+    lit_config.warning(f"Runtime dependency not found: {path}")
```

Comment:
Would the test fail without these files?

Reply:
Yes, all SwiftREPL tests fail without these files. Are you asking because it could also exit with an error? I am not exactly sure in which cases this Python code is executed. There might be a `--filter` parameter that excludes all SwiftREPL tests, but this code might still run. That's why I think a warning is good.